packages/main/src/plugin/extension-loader.ts

```diff
@@ -1074,6 +1076,109 @@ export class ExtensionLoader {
       },
     };
 
+    const _navigate = (navigateRequest: NavigateRequest) => {
```

Comment:
```suggestion
    const navigateTo = (navigateRequest: NavigateRequest) => {
```
do we need to use _ prefix ? usually this prefix is more to say that we ignore a variable from parameter list